### ISSUE TITLE
Create basic game view

### ISSUE THREAD

**Issue by JanKaifer** [JanKaifer/cell-lab-computer#8]

**Comment by JanKaifer** [comment]
How are you doing on this one?
- I'm lazy to creatre another branche and dont wont to mess up your work so In case you have done something if you could please push more often.

**Comment by JanKaifer** [comment]
And another thing to discuss - This was ment to be version for computer but since we are doing web app..
The layout will be mobile first - like we make layout for phones and then somehow scale it for computers?

**Comment by NicOfNoomers** [comment]
I can make a dynamic layout that is viewable for computer screens and mobile so leave that on me!

**Comment by JanKaifer** [comment]
Yeah and another thing with that - for react is recommended using react-bootstrap or resctstrap instead of importing bootstrap. I installed react-bootstrap since it is more complete. (Assuming you use bootstrap for layout)

**Comment by NicOfNoomers** [comment]
I am not using already made layouts because we want to make a nice visualy game so everything i customly made I just need a weekend and I will have most of the design up and running I just need to figure out what there are we going with. Modern and Scientific or Scetchy and Cartoonish! What do you think about that? @JanKaifer

**Comment by JanKaifer** [comment]
I'm terrible at arts :D So I'll leave that to you. (Just not too childish cartoon please ;) )
And about that bootstrap thing - didn't ment design, but the grid system is usefull - and it has mechanics for different layouts for phones, tablets, notebooks and big screens. However I assume that you are going to make css stuff so thats up to you if you are gonna use it.

**Comment by JanKaifer** [comment]
If not please uninstall react-bootstrap package.
